src/tictactoe/game_runner.clj

```diff
@@ -121,24 +104,164 @@
 
 (defn ^:private chosen-game-options
   [game-options]
-  (print-message (game-choices-message))
-  (loop-and-print game-options))
+  (print-message "You've chosen: ")
+  (display-game-menu-options game-options))
 
 (defn ^:private menu
   [game-types board-sizes]
-  (print-message (welcome-message))
-  (let [game-choice (pick game-types (game-type-message))
-        board-size (pick board-sizes (board-type-message))
+  (print-message "Welcome to Tic Tac Toe!\n")
+  (let [game-choice (pick game-types "Pick a game type: ")
+        board-size (pick board-sizes "What size board would you like to play?")
         player-one-marker (pick-marker)
         player-two-marker (pick-marker player-one-marker)]
     (let [choices {:game (get game-types game-choice)
                    :board (get board-sizes board-size)
                    :player1 player-one-marker
                    :player2 player-two-marker}]
-  (chosen-game-options choices)
-  choices)))
+      (chosen-game-options choices)
+      choices)))
+
+(defn ^:private setup-players 
+  [choices]
+  [{:type :human :marker (choices :player1)} {:type :human :marker (choices :player2)}])
+
+(defn ^:private make-board 
+  [board-choice]
+  (if (= "3x3" board-choice)
+    (vec (take 9 (range)))
+    (vec (take 16 (range)))))
+
+(defn ^:private empty-space? [space]
+  (number? space))
+
+(defn ^:private available-spaces
+  [board]
+  (filter #(empty-space? %) board))
+
+(defn ^:private to-integer 
+  [choice]
+  (try
+    (Integer. (re-find #"^\d+$" choice))
+    (catch Exception e nil)))
+
+(defn ^:private place-move 
+  [spot marker board]
+  (assoc board spot marker))
+
+(defn ^:private get-human-move
+  [available-spaces]
+  (println "\nPlease pick an available cell: ")
+  (let [choice (to-integer (get-input))]
+    (if (and choice (some #(= choice %) available-spaces))
+      choice
+      (recur available-spaces))))
+
+(defn ^:private next-move
+  [current-player board]
+  (if (= (current-player :type) :human)
+    (get-human-move (available-spaces board))))
+
+(defn ^:private row-divider
+  [length]
+  (->> 
+    "--"
+    (repeat (* length length))
+    (apply str)))
+
+(defn ^:private board-dividers
+  [number grid-size]
+  (cond (>= number (dec (* grid-size grid-size))) 
+          nil
+        (= (dec grid-size) (mod number grid-size)) 
+          (str "\n"(row-divider grid-size)"\n")
+        :else "|"))
+
+(defn ^:private grid-size
+  [board]
+  (math/sqrt (count board)))
+
+(defn ^:private clear-screen
+  []
+  (print (str (char 27) "[2J"))
+  (print (str (char 27) "[;H")))
+
+(defn ^:private display-board 
+  [board]
+  (println (apply str (map #(str "  "(board %)"  " (board-dividers % (grid-size board)))
```

Comment:
Spacing here. Also, this is pretty hard to visually parse. I'd probably use a `let` with some self-documenting variable names.

Reply:
The 4x4 board needs some prettifying.